avm/res/compute/virtual-machine/main.bicep

```diff
@@ -174,6 +174,31 @@ param extensionCustomScriptConfig object = {
   fileData: []
 }
 
+@description('Optional. The configuration for the [Nvidia Gpu Driver Windows] extension. Must at least contain the ["enabled": true] property to be executed.')
+param extensionNvidiaGpuDriverWindows object = {
+  enabled: false
+}
+
+@description('Optional. The configuration for the [Host Pool Registration] extension. Must at least contain the ["enabled": true] property to be executed. Needs a managed identy.')
+param extensionHostPoolRegistration object = {
+  enabled: false
+}
+
+@description('Optional. Required if host pool registiration extension is used. Name of the host pool.')
+param hostPoolName string = ''
```

Comment:
Why not make this part of the host pool registration object?  Name question for the other property

Reply:
done